controller/cache/info.go

```diff
@@ -74,6 +79,22 @@ func getIngress(un *unstructured.Unstructured) []v1.LoadBalancerIngress {
 	return res
 }
 
+func getRoute(un *unstructured.Unstructured) []v1.LoadBalancerIngress {
+	route, ok, err := unstructured.NestedSlice(un.Object, "status", "ingress")
+	if !ok || err != nil {
+		return nil
+	}
+	res := make([]v1.LoadBalancerIngress, 0)
+	for _, item := range route {
+		if lbIngress, ok := item.(map[string]interface{}); ok {
+			if hostname := lbIngress["routerCanonicalHostname"]; hostname != nil {
```

Comment:
`routerCanonicalHostname` is optional.   Please see https://docs.openshift.com/container-platform/4.6/rest_api/network_apis/route-route-openshift-io-v1.html